### PR TITLE
feat: (KW-158) Add kGuilder notification

### PR DESCRIPTION
**Pull request by Nashmod**
### Description

This PR adds a notification to explain to the user how to acquire cGuilder at the event.

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### How others should test

_Does this need to be tested by QA in the next release cycle? If so please give a brief explanation of how to test these changes._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

**Comment by shottah** [comment]
Can you post a screen recording and description of the change to be expected when testing.

**Comment by Nashmod** [comment]
> Can you post a screen recording and description of the change to be expected when testing.

Sure!

**Comment by Nashmod** [comment]
When testing you will see a new *kolektivoNotification*. You can click on it and see some reasons why to purchase kGuilder and what you can do with it. When you press done the notification will be permanently removed from the notifications stack.

https://user-images.githubusercontent.com/12153793/193376108-a3055633-424b-409f-89cb-302f3f14a038.mp4

**Comment by shottah** [comment]
Initial remarks (pre-review)
* When a user goes through these screens once, a flag should be set so that if they ever click that notification again, they won't need to see all screens, (there is code already in valora that considers this experience)
* User should be able to navigate from drawer menu, since the notification will be popped afterwards
* The final screen should say more (i'll push an update to this branch with more text)
* The final screen should have a button that opens up the "Receive" flow (since the user is receiving from the on-ramp agent)

**Comment by Nashmod** [comment]
@shottah I agree with all points except the first, If I'm not mistaken all notifications in this section only display once, so once completed they arent ever visible anymore. So there is no way to reopen them. Ofcourse I don't mind having it the other way but then how do you envision the notification being reopened? where should it also be available or should it neverbe removed from the top section? not even after the initial viewing?

File: src/home/NotificationBox.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ import Logger from 'src/utils/Logger'
 
 const TAG = 'NotificationBox'
 // Priority of static notifications
+const KOLEKTIVO_NOTIFICATTION_PRIORITY = 1100
```
Comment (shottah) [inline src/home/NotificationBox.tsx]: This is not higher priority that backup.

**Review by shottah — CHANGES_REQUESTED** [review]
This logic is implemented in the `home` saga but is specific to a core token of Kolektivo.
1. Move state information to `goldToken` reducer
2. Move actions and action types to `goldToken` actions

The only reason `idToNotification` is used in `home` reducer is because they are for `extraNotifications` that are all skippable and not rooted in a user experience.

File: src/home/NotificationBox.tsx (hunk 1 above)
Comment (Nashmod) [reply src/home/NotificationBox.tsx]: fixed

**Review by shottah — APPROVED** [review]

**Comment by shottah** [comment]
let er rip

**Pull request merged**